docs/components/concepts/workflow-patterns.md

```diff
@@ -118,19 +118,19 @@ The parallel multiple instance marker defines that this subprocess is executed m
 
 A typical situation is that a process needs to wait for some event to happen, e.g. some time to pass or some external message to arrive. This is related to [Workflow Pattern 23: Transient Trigger](http://www.workflowpatterns.com/patterns/control/new/wcp23.php).
 
-In BPMN, this is implemented using [events](/docs/components/modeler/bpmn/events/) (or [receive tasks](/docs/components/modeler/bpmn/receive-tasks/)):
+In BPMN, this is implemented using [events](/components/modeler/bpmn/events.md) (or [receive tasks](/components/modeler/bpmn/receive-tasks/receive-tasks.md)):
 
 <div bpmn="workflow-patterns/wait.bpmn" callouts="timer, message" />
 
 <span className="callout">1</span>
 
-The timer event causes the process to wait, in this case until a specific point in time is due or some duration has elapsed. Refer to [timer events](/docs/components/modeler/bpmn/timer-events/) for more details.
+The timer event causes the process to wait, in this case until a specific point in time is due or some duration has elapsed. Refer to [timer events](/components/modeler/bpmn/timer-events/timer-events.md) for more details.
 
 <span className="callout">2</span>
 
-The process will wait for a message to arrive. The message is an external trigger provided by API and can technically be anything, from a callback (e.g. via REST), over real messaging (like AMQP), or to notifications within your system. Refer to [message events](/docs/components/modeler/bpmn/message-events/) for more details.
+The process will wait for a message to arrive. The message is an external trigger provided by API and can technically be anything, from a callback (e.g. via REST), over real messaging (like AMQP), or to notifications within your system. Refer to [message events](/components/modeler/bpmn/message-events/message-events.md) for more details.
 
-You can read more about events in [our BPMN primer: events - waiting for something to happen](/docs/components/modeler/bpmn/bpmn-primer/#events-waiting-for-something-to-happen).
+You can read more about events in [our BPMN primer: events - waiting for something to happen](/components/modeler/bpmn/bpmn-primer/#events-waiting-for-something-to-happen).
```

Comment:
```suggestion
You can read more about events in [our BPMN primer: events - waiting for something to happen](/components/modeler/bpmn/bpmn-primer.md/#events-waiting-for-something-to-happen).
```

Reply:
I could have sworn I caught all of them, but thanks for this @pepopowitz! 😅